AdminUi/src/AdminUi/Controllers/LogsController.cs

```diff
@@ -0,0 +1,81 @@
+﻿using Backbone.Modules.Devices.Application;
+using Enmeshed.BuildingBlocks.API.Mvc;
+using Enmeshed.BuildingBlocks.API.Mvc.ControllerAttributes;
+using Enmeshed.BuildingBlocks.Application.Abstractions.Exceptions;
+using MediatR;
+using Microsoft.AspNetCore.Authorization;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.Extensions.Options;
+using ApplicationException = Enmeshed.BuildingBlocks.Application.Abstractions.Exceptions.ApplicationException;
+
+namespace AdminUi.Controllers;
+
+[Route("api/v1/[controller]")]
+[Authorize("ApiKey")]
+public class LogsController : ApiControllerBase
+{
+    private readonly ApplicationOptions _options;
+    private readonly ILoggerFactory _logger;
+
+    public LogsController(
+        IMediator mediator, IOptions<ApplicationOptions> options, ILoggerFactory logger) : base(mediator)
+    {
+        _options = options.Value;
+        _logger = logger;
+    }
+
+    [HttpPost]
+    [ProducesResponseType(StatusCodes.Status204NoContent)]
+    [ProducesError(StatusCodes.Status400BadRequest)]
+    public IActionResult CreateLog([FromBody] LogRequest request)
```

Comment:
```suggestion
    public IActionResult CreateLog(LogRequest request)
```

It should automatically be read from the body.